playwright.config.ts

```diff
@@ -0,0 +1,51 @@
+import { devices, PlaywrightTestConfig } from '@playwright/test';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+// See https://playwright.dev/docs/test-configuration.
+const config: PlaywrightTestConfig = {
+  testDir: './e2e/specs',
+  timeout: 3 * 60 * 1000,
+  expect: {
+    timeout: 40 * 1000,
+  },
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 1,
+  reporter: 'html',
+  globalSetup: require.resolve('./e2e/core/global-setup'),
+  webServer: {
+    command: 'yarn start',
+    url: process.env.E2E_UI_BASE_URL,
+    timeout: 120 * 1000,
+    reuseExistingServer: !process.env.CI,
+  },
```

Comment:
@denniskigen, I believe we cannot utilize this option as we have separate jobs for PRs and push commits, each with its own unique start command. This is due to the fact that we are running against a Docker container during the push.

Reply:
Do you still wanna change this? Because it's very close to getting merged.